hub/core/tensor.py

```diff
@@ -356,3 +389,12 @@ def __ixor__(self, other):
 
     def __ior__(self, other):
         raise NotImplementedError
+
+
+def load_tensor_meta(tensor_key: str, cache: LRUCache):
```

Comment:
is this being used anywhere?

Reply:
Nope. This was from the previous implementation of hashlist as meta files. I'll remove it.